File: Models/Knjiga.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Models{

      [Table("Knjiga")]
    public class Knjiga
    {
        [Key]
        public int ID { get; set; }


        [Required]
        public int Sifra { get; set; }


        [Required]
        [MaxLength(50)]
        public string Naziv { get; set; }

        public virtual Biblioteka Biblioteka{ get; set; }

        public virtual Zanr Zanr{ get; set; }
        
        [JsonIgnore]
        public virtual List<Iznajmljivanje> KnjigaClan { get; set; }

    }
}

File: Client/Knjiga.js
export class Knjiga{

    constructor(sifra, naziv, zanr, biblioteka){
        this.sifra=sifra;
        this.naziv=naziv;
        this.zanr=zanr;
        this.biblioteka=biblioteka;
    }

    crtaj(host){
       
        var tr = document.createElement("tr");
        host.appendChild(tr);

        var el = document.createElement("td");
        el.innerHTML=this.sifra;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML=this.naziv;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML=this.zanr;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML=this.biblioteka;
        tr.appendChild(el);
    }
}